### PR TITLE
Bridging feature!

### PR DESCRIPTION
**Pull request by JaylinYu**

### CODE REVIEW

File: nanolib/conf.c [hunk 1]
```diff
@@ -240,30 +243,39 @@ conf_init(conf **nanomq_conf)
 	(*nanomq_conf)->http_server.password = NULL;
 	(*nanomq_conf)->websocket.enable     = true;
 	(*nanomq_conf)->websocket.url        = NULL;
+	(*nanomq_conf)->bridge.bridge_mode   = false;
+	(*nanomq_conf)->bridge.sub_count     = 0;
+	(*nanomq_conf)->bridge.parallel      = 4;
```
Comment (JaylinYu) [inline nanolib/conf.c:248]: make it configurable

File: nanolib/conf.c [hunk 2]
```diff
@@ -240,30 +243,39 @@ conf_init(conf **nanomq_conf)
...
+	fprintf(stdout, "property_size:            %d\n",
+	    nanomq_conf->property_size);
+	fprintf(
+	    stdout, "msq_len:                  %d\n", nanomq_conf->msq_len);
+	fprintf(stdout, "qos_duration:             %d\n",
+	    nanomq_conf->qos_duration);
+	fprintf(stdout, "enable http server:       %s\n",
+	    nanomq_conf->http_server.enable ? "true" : "false");
+	fprintf(stdout, "http server port:         %d\n",
+	    nanomq_conf->http_server.port);
```
Comment (JaylinYu) [inline nanolib/conf.c:278]: please make it silent.

Comment (JaylinYu) [reply nanolib/conf.c:278]: and keep -url feature

File: nanolib/include/conf.h [hunk 3]
```diff
@@ -16,9 +16,13 @@
 	"{start|restart [-conf <path>] [-url <url>] [-daemon] [-tq_thread "   \
 	"<num>] [-max_tq_thread <num>] [-parallel <num>] [-qos_duration "     \
 	"<num>]}|stop}\n"                                                     \
-	"  -conf <path>          the path of a specified configuration file " \
+	"  -conf <path>          the path of a specified nanomq "             \
+	"configuration file "                                                 \
 	"\n"                                                                  \
-	"  -url <url>            the format of 'tcp://ip_addr:host'\n"        \
```
Comment (JaylinYu) [inline nanolib/include/conf.h:21]: please keep url feature

Comment (JaylinYu) [reply nanolib/include/conf.h:21]: and reminder

Comment (alvin1221) [reply nanolib/include/conf.h:21]: It was moved to conf.h:24 :"  -url <url>            the format of 'broker+tcp://ip_addr:host' "

**Review by alvin1221 — APPROVED** [review]

**Pull request merged**